### DISCUSSION TITLE
Windows 11 22H2: BSOD on install

### DISCUSSION THREAD

**Discussion by Emralegna** [tnodir/fort#40] [Q&A] [answered]
Hello,

I’ve got a BSOD PAGE FAULT IN NONPAGED AREA during the install, a bit after the cmd prompt shows up. I don’t have time to read what is written in there before the BSOD occurs.
I wondered if you can help me how can I troubleshoot to know what could be the cause of this? What do you need me to provide? Is there some logs that could help?

Thanks.

**Comment by tnodir** [comment]
Which version of Fort Firewall are you installing, x86 or x64?
On which version/build of Windows, x86 or x64?
Please provide memory crash dump file and screenshot of BSOD.

**Comment by bluelayer** [comment]
Hi!

I suspect it's the same like me, new windows 11 22h2.

**Comment by tnodir** [comment]
Let me check the new windows 11 22h2.

**Comment by bluelayer** [comment]
at the bsod moment i had pagefile disabled, but i can enable it if you need the minidump

**Comment by tnodir** [comment]
Please attach screenshot of BSOD for now.

I'll install the Win 11 22H2 to try it in VM.
Downloading "en-us_windows_11_business_editions_version_22h2_x64_dvd_17a08ce3.iso".

**Comment by bluelayer** [comment]
![photo_2022-09-23_11-05-04_2](https://user-images.githubusercontent.com/3768436/191938333-8f4208ed-8142-45cc-84c1-49f664be9dce.jpg)
just like he said,  PAGE FAULT IN NONPAGED AREA

**Comment by tnodir** [comment, marked as answer]
Please turn off the [HVCI (Core Isolation)](https://www.tomshardware.com/how-to/disable-vbs-windows-11).

Fort Firewall is not compatible with HVCI, because driver loads code in runtime to bypass EV Certificate restrictions.

**Reply by bluelayer** [reply]
ahhhhh I fell bad now, I had this disabled but I forgot to do it again grr

**Comment by tnodir** [comment]
I should had to add a check for HVCI in the driver loader.
My EV certificate is outdated now, so I can't sign a new driver by MS.
And I don't want to re-new the EV cert.

At least, I'll add a check to installer..

**Reply by bluelayer** [reply]
" a check to installer" sounds perfect.

**Comment by tnodir** [comment]
Also see https://github.com/tnodir/fort/discussions/108

**Reply by eabase** [reply]
Please consider my comment on 108. 
- https://github.com/tnodir/fort/discussions/108#discussioncomment-15561416
